### PR TITLE
Set OO_CHANNEL for telco-bastion workflow

### PR DESCRIPTION
**Pull request by johnbieren**
Allows passing the desired ocp version to kcli

Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

**Comment by johnbieren** [comment]
@yuvalk Can you review this? It allows us to set the openshift image in the ci-operator config file

File: ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh [hunk 1]
```diff
@@ -17,6 +17,11 @@ cat << EOF > ~/inventory
 sshd.bastion-telco ansible_ssh_user=tester ansible_ssh_common_args="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" ansible_password=$SSH_PASS
 EOF
 
+KCLI_PARAM=""
+if [ ! -z $OO_CHANNEL ] ; then
+    KCLI_PARAM="-P openshift_image=registry.svc.ci.openshift.org/ocp/release:$OO_CHANNEL"
```
Comment (yuvalk) [inline ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh:22]: contemplating if it'll be better to include that inside the kcli_parametrs.yml
and use envsubst to replace with real value.
then you can have the default dictated from here.

File: ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh [hunk 2]
```diff
@@ -31,8 +36,9 @@ cat << EOF > ~/ocp-install.yml
     retries: 5
   - name: Remove last run
     shell: kcli delete plan --yes upstream_ci
+    ignore_errors: yes
```
Comment (yuvalk) [inline ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh:39]: +1 on this
but seems unrelated to this commit.
it is better for git housekeeping in and clean history if this is a separate commit (can still go in same PR)

Comment (johnbieren) [reply ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh:39]: Fixed

File: ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh (hunk 1 above)
Comment (johnbieren) [reply ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh:22]: I can put an else here and set the image to some default image if you'd like
